### DISCUSSION TITLE
How to make a doc only with *.stories.tsx - without a story

### DISCUSSION THREAD

**Discussion by sara-tapusoa** [storybookjs/storybook#28261] [Help] [answered]
### Summary

Since `.mdx` has been deprecated I have had a hard time trying to make documentation without stories. I don't want to hide all stories in my design system, just a few that don't need stories. How do I do this?

### Additional information

import type { StorybookConfig } from '@storybook/react-vite'

const config: StorybookConfig = {
  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
  addons: [
    '@storybook/addon-links',
    '@storybook/addon-essentials',
    '@storybook/addon-interactions',
    '@storybook/addon-mdx-gfm',
  ],
  framework: {
    name: '@storybook/react-vite',
    options: {},
  },
  docs: {
    autodocs: true,
  },
  typescript: {
    reactDocgen: 'react-docgen-typescript',
  },
}
export default config
// ignore the mdx configurations since it has been deprecated

// my *.stories.tsx file:
...
const meta: Meta<typeof Button> = {
  title: 'Shadows',
  component: Button,
  parameters: {
    docs: {
      page: ButtonDocsPage,
      description: {
        component: `
This documentation details the shadow levels used across the design system. Each level serves a specific visual function:
        `,
      },
    },
  },
}

{
  /*
    controls: { hideNoControlsWarning: true },
    previewTabs: {
      canvas: { hidden: true },
    },
    includeStories: [],
  */
}

export default meta

export const Filled: Story = {
  args: {
    children: 'Filled',
    ...
  },
}

{
  /*
parameters: {
    docsOnly: true
  }
  */
}

// I tried some of these commented out solutions but it didn't accomplish what I wanted - I just want a documentation only (no story associated with it)

**Comment by bzh2610** [comment, marked as answer]
Hi,

From my understanding, `.stories.mdx` were deprecated, but you can still write `.mdx`files.

```
import { Meta } from "@storybook/blocks";

<Meta title="Some test/Test" />

## A test doc

Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

<img width="866" alt="Screenshot 2024-06-19 at 10 44 36" src="https://github.com/storybookjs/storybook/assets/4430790/0edba140-2dc5-4abd-a543-32c22021f089">

**Comment by rdevries** [comment]
So, what happens when it's no longer deprecated? what is the solution then?
